orchestrator/src/test/java/com/scottlogic/deg/orchestrator/endtoend/JarExecuteTests.java

```diff
@@ -0,0 +1,28 @@
+package com.scottlogic.deg.orchestrator.endtoend;
+
+import org.junit.jupiter.api.Test;
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class JarExecuteTests {
+
+    @Test
+    void GenerateSuccessfullyFromJar() throws Exception {
+            ProcessBuilder pb = new ProcessBuilder("java", "-jar", "build/libs/generator.jar", "generate","-p=src/test/java/com/scottlogic/deg/orchestrator/endtoend/testprofile.profile.json","--max-rows=1");
+            Process p = pb.start();
+            BufferedReader BufferedSTDOUTReader = new BufferedReader(new InputStreamReader(p.getInputStream()));
+            List<String> STDOUT = new ArrayList<String>();
```

Comment:
I don't really like the name of the variable here - all caps should only be used for constants.  A better name would be something like `collectedOutput`